models/evc.py

```diff
@@ -1183,6 +1185,7 @@ def check_trace(
                                     ) is False:
                 log.warning(f"Invalid trace from uni_z: {trace_z}")
                 circuits_checked[circuit_id] = False
+        return True
```

Comment:
Returning true is not helpful for this function since the value is already updated on the parameter `circuits_checked`. I would suggest not add this return

Reply:
The idea of returning a boolean here was to know if `circuit_id` was added to `circuits_checked` or not. It is different from `circuits_checked[circuit_id]`. In fact, if circuits_checked[circuit_id] is set to False, the function will return True. Anyway, it is not necessary and can be removed.